src/includes/class-alma-wc-refund.php

```diff
@@ -144,7 +144,7 @@ public function gettext( $translation, $text, $domain ) {
 		}
 
 		if ( count( $this->admin_texts_to_change ) === $this->number_of_texts_changed ) {
-			remove_filter( 'gettext', array( $this, 'gettext' ), 10 );
+			remove_filter( 'gettext', array( $this, 'gettext' ) );
```

Comment:
The parameter "priority" default value is "10", but as I forced it in the 
`add_filter( 'gettext', array( $this, 'gettext' ), 10, 3 );`
filter, that looks more suitable to pass it in `remove_filter()` function.

I changed this in my last commit.

Reply:
I had removed this priority because it is the same than default one, it is not mandatory to write it, and it removes a notice in phpstorm if you don't write it... anyway it is not so important